liquidity-amo/contracts/V2AMO.sol

```diff
@@ -342,14 +402,12 @@ contract SolidlyV2AMO is ISolidlyV2AMO, MasterAMO {
     function _mintSellFarm() internal override returns (uint256 liquidity, uint256 newBoostPrice) {
         (uint256 boostReserve, uint256 usdReserve) = getReserves();
 
-        uint256 boostAmountIn = (((usdReserve - boostReserve) / 2) * boostSellRatio) / FACTOR;
+        uint256 boostAmountIn = ((sqrt(usdReserve * boostReserve) - boostReserve) * boostSellRatio) / FACTOR;
```

Comment:
We should also adjust the math formula in `_unfarmBuyBurn()` when calculating `usdNeeded`.